From 8414ffd8dcd310616ec4f0eb6fdd4d06d15f11db Mon Sep 17 00:00:00 2001
From: Tanvi Chauhan <tanvi.chauhan@einfochips.com>
Date: Thu, 10 Feb 2022 12:34:21 +0530
Subject: [PATCH 4/5] setup: meta-einfochips-ap1302 layer is added for AP1302

append new mete-einfochips-layer to support AP1302 interface
with thor96 board

diff --git a/tools/imx-setup-release.sh b/tools/imx-setup-release.sh
index 812c99c60..aaa7e7873 100755
--- a/tools/imx-setup-release.sh
+++ b/tools/imx-setup-release.sh
@@ -197,6 +197,9 @@ if [ -d ../sources/meta-ivi ]; then
     echo "BBLAYERS += \"\${BSPDIR}/sources/meta-ivi/meta-ivi-test\"" >> $BUILD_DIR/conf/bblayers.conf
 fi
 
+echo "" >> conf/bblayers.conf
+echo "BBLAYERS += \"\${BSPDIR}/sources/meta-einfochips-ap1302\"" >> conf/bblayers.conf
+
 echo BSPDIR=$BSPDIR
 echo BUILD_DIR=$BUILD_DIR
 
-- 
2.17.1

